opencti-platform/opencti-front/src/schema/relay.schema.graphql

```diff
@@ -12327,6 +12328,12 @@ type ExclusionListEdge {
   node: ExclusionList!
 }
 
+type ExclusionListCacheStatus {
+  refreshAskDate: String!
+  cacheDate: String!
+  allNodeCacheUpdated: Boolean!
```

Comment:
Why String for the dates and not DateTime ? and also all these values can be null I guess ?
Also the naming is not ideal, here are my suggestions if you like them better:
currentCacheDate: DateTime
isCacheRebuildInProgress: Boolean!

So that the frontend can only use this isCacheRebuildInProgress to display a progress status if it's true, and if it's false, then cache is up to date. Do we need more info ?

Reply:
or maybe we would like the refreshAskDate for debug purposes ?

Reply:
That's what I was thinking, that refreshAskDate could still be useful sometimes to troubleshoot a cache problem if needed. For the types as String instead of DateTime, I didn't want to be too restrictive. I know that we sometimes talked about using version numbers instead of dates for the cache handling, so typing them as String gives us some leeway on the API side. Maybe the change could be to rename 'cacheDate' to 'cacheVersion'? This way it is more generic